### DISCUSSION TITLE
Combine command palettes into one, and change hotkey

### DISCUSSION THREAD

**Discussion by cobrexus** [community/community#7952] [Other Feature Feedback, Questions, & Ideas] [answered]
My suggestion is to change the command completely to <kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <kbd>P</kbd> and <kbd>F1</kbd> to open command palette so as to mach VS Code.

There are actually two commands, <kbd>Ctrl</kbd> + <kbd>K</kbd> or <kbd>Ctrl</kbd> + <kbd>Alt</kbd> + <kbd>K</kbd> to open command palette (Search or jump to...), and <kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <kbd>K</kbd> to open command palette in command mode (Run a command).

It doesn't make sense for there to be two command palettes, since now I have to remember which commands work in which command palette. All of these actions should just be in one <kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <kbd>P</kbd> / <kbd>F1</kbd> command palette. Make two sections if you will, but please only one command palette.

And currently, there is no hotkey to open file navigation, so I have to click on <kbd>Go to file</kbd>. It would be nice if <kbd>Ctrl</kbd> + <kbd>P</kbd> did this just like in VS Code.

**Comment by gaknoll** [comment, marked as answer]
Hi @AryanBeezadhur, we just shipped a new feature that allows you to customize the keyboard shortcuts used to open the command palette. You can edit your shortcuts in the [Accessibility section](https://github.com/settings/accessibility) of your user settings. `Ctrl + Shift + P` is one of our supported options! 

Regarding your feedback on being required to use modes to use commands, we are currently working towards a release that will allow you to access commands from "both" command palettes.

**Reply by aheckler** [reply]
It's possible to try to set both shortcuts to the same thing, which gives a rather vague error:

<img width="889" alt="Screenshot taken on 2022-02-03 at 18 04 06 UTC@2x" src="https://user-images.githubusercontent.com/5512652/152402130-30261ccf-37f8-4100-b3f3-cf8f9d191e08.png">
